index.html

```diff
@@ -275,15 +275,16 @@ <h1>About Myself</h1>
         <h2 class="contact-title">Contact me</h2>
         <p class="contact-desc">If you have an application you are interested in developing, a feature that you need
             built or a project that needs coding. I’d love to help with it </p>
-        <form action="https://formspree.io/f/mrgdbjnk" method="post">
-            <input type="text" name="name" maxlength="30" placeholder="Full name" class="form" required>
+        <form id="form" action="https://formspree.io/f/mrgdbjnk" method="post">
+            <input type="text" name="name" maxlength="30" id="name" placeholder="Full name" class="form" required>
 
-            <input type="email" name="email" placeholder="Email address" class="form" required>
+            <input  name="email" id="email" placeholder="Email address" class="form" required>
 
```

Comment:
- [ ] Email field containing uppercase characters should display an error message but currently, I am able to send Uppercase characters in the email please address that